plugins/ufm_log_analyzer_plugin/unit_tests/test_ibdiagnet_log_analyzer.py

```diff
@@ -0,0 +1,46 @@
+"""
+@copyright:
+    Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+
+    This software product is a proprietary product of Nvidia Corporation and its affiliates
+    (the "Company") and all right, title, and interest in and to the
+    software product, including all associated intellectual property rights,
+    are and shall remain exclusively with the Company.
+
+    This software product is governed by the End User License Agreement
+    provided with the software product.
+
+@author: Miryam Schwartz
+@date:   Dec 08, 2024
+"""
+
+import unittest
+import sys
+import os
+
+sys.path.append(os.getcwd())
+sys.path.append("/".join(os.getcwd().split("/")[:-1]))
+sys.path.append("/".join(os.getcwd().split("/")[:-1]) + "/src")
```

Comment:
Why is it needed?

Reply:
I get import error without it

Reply:
Adding this to every file that implements a test is not fun ;)

If needed extra configuration, you should add a project.toml file or a pytest.ini
(since this project already has a file for the pylint, I would add another one for the pytest (pytest.ini)

Reply:
conftest.py file was added